e2e/update_snapshots_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
```

Comment:
is this new test getting run?
https://github.com/aspect-build/rules_jest/pull/73/checks doesn't show anything about snapshots

Reply:
Yes, it shows up as Additional Integration tests on the main WORKSPACE https://github.com/aspect-build/rules_jest/actions/runs/3643152320/jobs/6151085928